org.eclipse.corrosion/src/org/eclipse/corrosion/launch/RustLaunchDelegateTools.java

```diff
@@ -103,7 +103,7 @@ public static File convertToAbsolutePath(String path) {
 		if (filePath == null) {
 			return file;
 		}
-		return filePath.getRawLocation().toFile();
+		return new File(ResourcesPlugin.getWorkspace().getRoot().getRawLocation().toFile(), path);
```

Comment:
This can be source of bugs as the path of the project may not be related to workspace one.